commands_to_load/Mod.py

```diff
@@ -82,5 +83,375 @@ def __init__(self, bot):
         if eObj is not False:
             await ctx.send(embed=eObj)
 
+    @commands.command(aliases=['propm'])
+    async def propagatemute(self, ctx):
+        """Ensures all channels have the muted role as part of its permissions.""" 
+        ## Avoid the admin category of channels and rules and announcments channel since nobody can talk in there anyway
+
+        logger.info('[Mod propagatemute()] propagatemute function detected by user ' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod propagatemute()] invoking message deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod propagatemute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        # Get guild
+        channels = ctx.guild.channels
+
+        # Set up the perms overwrite
+        overwrite = discord.PermissionOverwrite()
+        setattr(overwrite, 'send_messages', False)
+        setattr(overwrite, 'manage_messages', False)
+        setattr(overwrite, 'manage_channels', False)
+        setattr(overwrite, 'manage_guild', False)
+        setattr(overwrite, 'manage_nicknames', False)
+        setattr(overwrite, 'manage_roles', False)
+        
+        # Loop through channels and change the perms
+        for channel in channels: 
+            if channel.id not in adminChannels:
+                await channel.set_permissions(MUTED_ROLE, overwrite=overwrite)
+
+        eObj = em(description='Muted permissions spread though all channels like herpies. Enjoy :)', footer='Moderator action')
+        await ctx.send(embed=eObj, delete_after=3.0)
+
+    @commands.command()
+    async def slowmode(self, ctx, time = 10): 
+        logger.info('[Mod slowmode()] slowmode function detected by user' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod slowmode()] invoking message deleted')
+
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod slowmode()] unathoriezed command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+         
+        await ctx.message.channel.edit(slowmode_delay=time, reason=None)
+        logger.info('[Mod slowmode()] slowmode enable on channel: ' + str(ctx.message.channel) + ', time between messages set to ' + str(time))
+
+    @commands.command()
+    async def makechannel(self, ctx, name, secret = 0):

```

Comment:
got this error when using it
2019-03-31 02:46:57 = WARNING -   File "/usr/src/app/commands_to_load/Mod.py", line 179, in makechannel
    ch = await ctx.guild.create_text_channel(name, overwrites=overwrite)

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/guild.py", line 670, in create_text_channel
    data = await self._create_channel(name, overwrites, ChannelType.text, category, reason=reason, **options)

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/guild.py", line 573, in _create_channel
    'id': target.id

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING - AttributeError: 'NoneType' object has no attribute 'id'

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING - 
The above exception was the direct cause of the following exception:


2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING - Traceback (most recent call last):

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/bot.py", line 898, in invoke
    await ctx.command.invoke(ctx)

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 611, in invoke
    await injected(*ctx.args, **ctx.kwargs)

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING -   File "/usr/local/lib/python3.5/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(exc) from exc

2019-03-31 02:46:57 = WARNING - 
2019-03-31 02:46:57 = WARNING - discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'id'

2019-03-31 02:46:57 = WARNING -

Reply:
(See my review for the cause I've found)